packages/open-next/src/build/copyTracedFiles.ts

```diff
@@ -203,25 +202,36 @@ File ${fullFilePath} does not exist
     }
   });
 
-  readdirSync(standaloneNextDir).forEach((f) => {
-    if (statSync(path.join(standaloneNextDir, f)).isDirectory()) return;
-    copyFileSync(path.join(standaloneNextDir, f), path.join(outputNextDir, f));
-  });
+  readdirSync(standaloneNextDir)
+    .filter(
+      (fileOrDir) =>
+        !statSync(path.join(standaloneNextDir, fileOrDir)).isDirectory(),
+    )
+    .forEach((file) =>
+      copyFileSync(
+        path.join(standaloneNextDir, file),
+        path.join(outputNextDir, file),
+      ),
+    );
 
   // We then need to copy all the files at the root of server
 
   mkdirSync(path.join(outputNextDir, "server"), { recursive: true });
 
-  readdirSync(path.join(standaloneNextDir, "server")).forEach((f) => {
-    if (statSync(path.join(standaloneNextDir, "server", f)).isDirectory())
-      return;
-    if (f !== "server.js") {
+  readdirSync(path.join(standaloneNextDir, "server"))
+    .filter(
```

Comment:
We could do it in 1 filter instead of 2, it avoids having to iterate over everything twice